src/network/networkd-network.c

```diff
@@ -205,6 +205,7 @@ static int network_load_one(Manager *manager, const char *filename) {
                               "Bridge\0"
                               "BridgeFDB\0"
                               "BridgeVLAN\0"
+                              "PrefixDelegation\0"
```

Comment:
IPv6PrefixDelegation so that it's in line with the previously defined IPv6Prefix section.